src/components/style/Filters.css

```diff
@@ -0,0 +1,32 @@
+
+#filters-holder img {
+    margin: 1em;
+}
```

Comment:
Here, a class name would be more suitable - again consider the case that multiple `<Filters />` are drawn to the page, or another component uses the ID `#filters-holder`.